### PR TITLE
[BAD-219] - Support for all BD services on EMR Hadoop tech

### PR DESCRIPTION
**Pull request by stepanovdg**
DO not merge until http://jira.pentaho.com/browse/COMP-1229 would be resolved.

### CODE REVIEW

**Comment by stepanovdg** [comment]
Component process is rejected. Pull-request updated. Now this could be merged.
@mattyb149 Could you please take a look ?

File: emr32/build.properties [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+dependency.pentaho-hadoop-snappy.revision=0.0.1
+dependency.pentaho-vfs-browser.revision=TRUNK-SNAPSHOT
+
+dependency.xstream.revision=1.4.2
+dependency.commons-lang.revision=2.6
+dependency.jackson.revision=2.5.1
+jackson.version=2.5.1
+dependency.amazon-sdk.revision=1.9.18
+dependency.thrift.revision=0.9.0
+dependency.lzo.revision=0.4.19
```
Comment (mattyb149) [inline emr32/build.properties]: This can/should be removed

File: emr32/build.properties [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+dependency.kettle.revision=TRUNK-SNAPSHOT
+dependency.pentaho-hdfs-vfs.revision=TRUNK-SNAPSHOT
+dependency.pentaho-hadoop-snappy.revision=0.0.1
+dependency.pentaho-vfs-browser.revision=TRUNK-SNAPSHOT
+
+dependency.xstream.revision=1.4.2
+dependency.commons-lang.revision=2.6
+dependency.jackson.revision=2.5.1
+jackson.version=2.5.1
+dependency.amazon-sdk.revision=1.9.18
```
Comment (mattyb149) [inline emr32/build.properties]: It doesn't appear this version of the Amazon SDK has been approved via the Component Process

**Comment by stepanovdg** [comment]
* hadoop.lzo.revision property - removed
* thrift property as libthrift dependencies - are required for hive and has already passed component process http://jira.pentaho.com/browse/COMP-286
* The only one jar fr which component process issue was committed - was hadoop-lzo.jar. The main reason was that it was not able to resolve through public mvn repositories and it should be added to Pentaho nexus (at that moment). As this shim is still in modification (s3 purposes) and list of required jars is not stable - component process for all other libraries still wasn`t submitted. So will check all components.

**Comment by mattyb149** [comment]
We should regression test at least one other shim due to the common FileSystemProxy change

**Pull request merged**